lib/sisimai/bite/email/exchange2007.rb

```diff
@@ -0,0 +1,178 @@
+module Sisimai::Bite::Email
+  module Exchange2007
+    # Sisimai::Bite::Email::Exchange2007 parses a bounce email which created by
+    # Microsoft Exchange Server 2007.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Exchange2007.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'subject'          => %r/\AUndeliverable:/,
+        :'content-language' => %r/\A[a-z]{2}(?:[-][A-Z]{2})?\z/,
+      }
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->